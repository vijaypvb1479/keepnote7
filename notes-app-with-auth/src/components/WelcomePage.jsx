import React, { Fragment } from 'react';
import welcome from '../images/welcome.jpg';
import { withStyles } from '@material-ui/core/styles';


const styles = theme => ({
    img: {
        display: 'block',
        marginLeft: 'auto',
        marginRight: 'auto',
        marginTop: theme.spacing.unit * 3,
        width: '40%',
        [theme.breakpoints.down('sm')]: {
            width: '60%',
        },
    },
});

const WelcomePage = (props) => (
    <Fragment>

    <img src={welcome} alt="welcome" className={props.classes.img} />
    </Fragment>
);

export default withStyles(styles)(WelcomePage);