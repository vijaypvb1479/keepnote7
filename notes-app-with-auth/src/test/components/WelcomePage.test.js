import React from 'react';
import {shallow } from 'enzyme';
import WelcomePage from '../../components/WelcomePage';

import {createSerializer} from 'enzyme-to-json';
 
expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

test('test welcome page', () => {

    const wrapper = shallow(<WelcomePage />);
    expect(wrapper).toMatchSnapshot();


});