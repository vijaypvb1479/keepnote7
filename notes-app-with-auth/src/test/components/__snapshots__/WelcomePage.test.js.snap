// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test welcome page 1`] = `
<WelcomePage
  classes={
    Object {
      "img": "WelcomePage-img-1",
    }
  }
/>
`;
